tests/unittests/regression/test_kendall.py

```diff
@@ -0,0 +1,124 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import operator
+from collections import namedtuple
+from functools import partial
+
+import pytest
+import torch
+from scipy.stats import kendalltau
+
+from torchmetrics.functional.regression.kendall import kendall_rank_corrcoef
+from torchmetrics.regression.kendall import KendallRankCorrCoef
+from torchmetrics.utilities.imports import _compare_version
+from unittests.helpers import seed_all
+from unittests.helpers.testers import BATCH_SIZE, EXTRA_DIM, NUM_BATCHES, MetricTester
+
+_SCIPY_GREATER_EQUAL_1_8 = _compare_version("scipy", operator.ge, "1.8.0")
```

Comment:
Is there a reason we are not fixing scipy to be >=1.8.0?
Should not be a problem when only used for testing.